viewer-admin/src/main/webapp/WEB-INF/jsp/services/editfeaturetyperelation.jsp

```diff
@@ -105,6 +105,20 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                                 <td></td>
                                 <td></td>
                             </tr>
+                            <tr>
+                                <td>Zoek volgende relatie:</td>
+                                <td><stripes:checkbox name="relation.searchNextRelation"  checked="relation.searchNextRelation" /><br>
```

Comment:
hoe zit het met een upgraded versie van TM waar deze change niet in zit? de default waarde moet dan `true` zijn anders gaat er van alles stuk bij uograde

Reply:
De search next relation heeft ook een default value true.
De can create relation heeft een default value false en heeft geen impact op bestaande TM. Dit wordt niet gebruikt en is van toepassing op de feature-api.